QUEST_LV_0100.tsv

```diff
@@ -1960,21 +1960,21 @@ QUEST_LV_0100_20150317_001959	There are many artifacts here that praise the hero
-QUEST_LV_0100_20150317_001965	And when Lydia Schaffen got old, Agailla Flurry became the last disciple.{nl}As the other two disciples were, Agailla Flurry was a great person.
-QUEST_LV_0100_20150317_001966	She learned all magics that existed then.{nl}She also built Mage Tower. She was so amazing.
-QUEST_LV_0100_20150317_001967	Our old man always complains.{nl}Even if I give him the best things to him, he always complains.{nl}
-QUEST_LV_0100_20150317_001968	I don't wanna help him since he always complains.{nl}I am sorry, but can you tell my feelings to the old man?
-QUEST_LV_0100_20150317_001969	When he first came here, he had hard time adopting to the new environment.{nl}I used to cook him some food and made him some clothes..
-QUEST_LV_0100_20150317_001970	How come Cahill didn't come here and how come you are saying that?{nl}Well, it doesn't matter. Are you finished?
-QUEST_LV_0100_20150317_001971	Hey. Since you came, please do some errands.{nl}You see those lumbers over there? Please bring me ones that are fine among them.
-QUEST_LV_0100_20150317_001972	Cahill is helpful. {nl}Except for his complaints..
-QUEST_LV_0100_20150317_001973	What! Can't you find more decent ones?{nl}Well.. that's okay..
-QUEST_LV_0100_20150317_001974	Please give this crutches which I made myself to Cahill.{nl}Where was Cahill hurt his ankle..
-QUEST_LV_0100_20150317_001975	Before I change my mind, get outta here.
-QUEST_LV_0100_20150317_001976	Crutches from Moze..{nl}I really don't know what he is thinking.
-QUEST_LV_0100_20150317_001977	I didn't say it's over.{nl}Please get me the leather of Wing Frogs near this camp.{nl}
+QUEST_LV_0100_20150317_001963	$There was a priest called Maven in the past.{nl}He is the person who built the great cathedral.{nl}
+QUEST_LV_0100_20150317_001964	$Maven lived for a long time and had three disciples.{nl}The first one was Ruklys and the second one was Lydia Schaffen.{nl}
+QUEST_LV_0100_20150317_001965	$And when Lydia Schaffen grew old, Agailla Flurry became the last disciple.{nl}As the other two disciples were, Agailla Flurry was a great person.
+QUEST_LV_0100_20150317_001966	$She learned all magics that existed then.{nl}She also built Mage Tower. She must've been amazing.
+QUEST_LV_0100_20150317_001967	$Our old man always complains.{nl}Even when I give him the best things to him, he complains.{nl}
+QUEST_LV_0100_20150317_001968	$He always complains, so I don't want to help him.{nl}Sorry, but can you tell that old man there how I feel?
+QUEST_LV_0100_20150317_001969	$When he first came here, he had a hard time adapting to the new environment.{nl}I used to cook him food and make him clothes..
+QUEST_LV_0100_20150317_001970	$Why didn't Kayhill come here and why you saying that?{nl}Well, it doesn't matter. Are you finished?
+QUEST_LV_0100_20150317_001971	$Hey. Since you came, please do some errands.{nl}You see those lumbers over there? Please bring me the good ones among them.
```

Comment:
"Hey. Since you came, please do some errands for me.{nl}You see that pile of lumber over there? Bring me the good ones among them."
?